client/src/r_draw.cpp

```diff
@@ -454,44 +472,51 @@ void R_CopyTranslationRGB (int fromplayer, int toplayer)
 
 // [RH] Create a player's translation table based on
 //		a given mid-range color.
-void R_BuildPlayerTranslation(int player, argb_t dest_color)
+void R_BuildPlayerTranslation(int player, argb_t dest_color, int colorpreset)
```

Comment:
A default value for `colorpreset` would simplify this function's use.

Reply:
The default value could be the enum `NO_COLORPRESET` (or similar phrasing) mentioned below